README.md

```diff
@@ -227,6 +227,30 @@ export class GqlThrottlerGuard extends ThrottlerGuard {
 }
 ```
 
+However, when using Apollo Express/Fastify or Mercurius, it's important to configure the context correctly in the GraphQLModule to avoid any problems.
+
+#### Apollo Server (for Express):
+
+For Apollo Server running on Express, you can set up the context in your GraphQLModule configuration as follows:
+
+```typescript
+GraphQLModule.forRoot({
+  // ... other GraphQL module options
+  context: ({ req, res }) => ({ req, res }),
+});
+```
+
+#### Apollo Server (for Fastify) & Mercurius:
+
+When using Apollo Server with Fastify or Mercurius, you need to configure the context differently. You should use request and reply objects. Here's an example:
+
+```typescript
+GraphQLModule.forRoot({
+  // ... other GraphQL module options
+  context: (request, reply) => ({ request, reply }),
+})
+```
```

Comment:
I _think_ we need to set this up as
```suggestion
```typescript
GraphQLModule.forRoot({
  // ... other GraphQL module options
  context: (request, reply) => ({ req: request, res: reply }),
})
```
```
because we expect `req` and `res` in the guard. This of course could by modified though, by overriding the `getRequestResponse` method of the `ThrottlerGuard`

Reply:
I take it back, it wouldn't matter. That `getRequestResponse` has to be overridden either way, so calling it whatever (like `({ request ,reply })` is absolutely fine